src/Tokenizer/Transformer/BraceClassInstantiationTransformer.php

```diff
@@ -52,12 +52,20 @@ public function process(Tokens $tokens, Token $token, $index)
             return;
         }
 
-        $closeIndex = $tokens->findBlockEnd(Tokens::BLOCK_TYPE_PARENTHESIS_BRACE, $index);
-
-        if (!$tokens[$tokens->getNextMeaningfulToken($closeIndex)]->isGivenKind(array(T_OBJECT_OPERATOR, T_DOUBLE_COLON))) {
+        if ($tokens[$tokens->getPrevMeaningfulToken($index)]->equalsAny(array(
+            array(T_STRING),
+            ']',
+            array(CT::T_ARRAY_SQUARE_BRACE_CLOSE),
+            array(CT::T_ARRAY_INDEX_CURLY_BRACE_CLOSE),
```

Comment:
please defined the `getPriority` method in this class to ensure this transformer runs after the one(s) that transform to these two customs token types

Reply:
Done.